### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by guidotijskens**
pylint cleanup
fixed an issue showing up when an empty lvm is present on the on-premise workstation.
.
tested
images with lvm and without lvm, on workstation with lvm and without lvm

### CODE REVIEW

File: lib/oci_utils/impl/migrate/oci-image-migrate-import-main.py [hunk 1]
```diff
@@ -25,10 +25,7 @@
 from oci_utils.migrate import system_tools
 from oci_utils.migrate import terminal_dimension
 from oci_utils.migrate.exception import OciMigrateException
-from oci_utils.migrate.migrate_tools import get_config_data as get_config_data
-
-if sys.version_info.major < 3:
-    exit_with_msg('Python version 3 is a requirement to run this utility.')
+from oci_utils.migrate.migrate_tools import get_config_data
```
Comment (ejannett) [inline lib/oci_utils/impl/migrate/oci-image-migrate-import-main.py:29]: if 'ci' is kind of sub-category of "importation". better to use oci-utils.import.ci as logger name

File: lib/oci_utils/impl/migrate/oci-image-migrate-upload-main.py [hunk 2]
```diff
@@ -127,8 +130,8 @@ def main():
     try:
         #
         # The oci configuration.
-        oci_config = migrate_tools.get_oci_config()
-        _logger.debug('Found oci config file')
+        # oci_config = migrate_tools.get_oci_config()
```
Comment (ejannett) [inline lib/oci_utils/impl/migrate/oci-image-migrate-upload-main.py:133]: if this is not used , juste remove it

File: lib/oci_utils/migrate/__init__.py [hunk 3]
```diff
@@ -14,6 +17,10 @@
 
 import yaml
 
+from ..migrate import migrate_data
+
+_logger = logging.getLogger('__name__')
```
Comment (ejannett) [inline lib/oci_utils/migrate/__init__.py:22]: keep conventional naming : oci_utils.migrate

File: lib/oci_utils/migrate/__init__.py [hunk 4]
```diff
@@ -166,18 +229,16 @@ def terminal_dimension():
...
     try:
-        # rowstr, colstr = os.popen('stty size', 'r').read().split()
-        # return int(rowstr), int(colstr)
         terminal_size = os.get_terminal_size()
         return terminal_size.lines, terminal_size.columns
-    except Exception as e:
+    except Exception:
         #
         # fail to get terminal dimension, because not connected to terminal?
         # returning dummy
```
Comment (ejannett) [inline lib/oci_utils/migrate/__init__.py:237]: shall we log a debug message ?

File: lib/oci_utils/migrate/__init__.py [hunk 5]
```diff
@@ -209,6 +271,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         """
         OciMigrateConfParam cleanup and exit.
         """
+        # placeholder
```
Comment (ejannett) [inline lib/oci_utils/migrate/__init__.py:274]: if this is supposed to be overwritten by sub classes implementation and that's is 
mandatory , think about raising an exception or an assert

File: lib/oci_utils/migrate/image_types/qcow2.py [hunk 6]
```diff
@@ -90,19 +91,17 @@ def __init__(self, filename):
         filename: str
             Full path of the qcow2 image file.
         """
-        _logger.debug('qcow2 header size: %d bytes' % self.head_size)
-        super(Qcow2Head, self).__init__(filename)
+        _logger.debug('qcow2 header size: %d bytes', self.head_size)
+        super().__init__(filename)
```
Comment (ejannett) [inline lib/oci_utils/migrate/image_types/qcow2.py:95]: call to super methods should be the first calls

File: lib/oci_utils/migrate/image_types/vmdk.py [hunk 7]
```diff
@@ -277,42 +265,35 @@ def type_specific_prereq_test(self):
...
-                       % (self.img_header['desc']['createType'],
-                          prereqs['vmdk_supported_types'])
+                       % (self.img_header['desc']['createType'], prereqs['vmdk_supported_types'])
             #
             # Warning, for now, streamOptimized format will probably cause problems.
             if self.img_header['desc']['createType'] == 'streamOptimized':
-                _ = read_yn('  %s\n  Continue' % self.streamoptimized_msg,
-                            yn=False,
-                            suppose_yes=migrate_data.yes_flag)
+                _ = read_yn('  %s\n  Continue' % self.streamoptimized_msg, yn=False, suppose_yes=migrate_data.yes_flag)
```
Comment (ejannett) [inline lib/oci_utils/migrate/image_types/vmdk.py:297]: why are we not using the returned answer ?

File: lib/oci_utils/migrate/imgdevice.py [hunk 8]
```diff
@@ -707,56 +636,49 @@ def remount_partitions(self):
...
         # non-root partitions and mount those on the root partition.
         # The list is sorted to avoid overwriting subdirectory mounts like
         # /var, /var/log, /van/log/auto,.....
         mountlist = []
-        for k, v in list(self._img_info['partitions'].items()):
-            _logger.debug('remount?? %s' % k)
-            _logger.debug('remount?? %s' % v)
+        for k, v in list(self.image_info['partitions'].items()):
+            _logger.debug('remount?? %s', k)
+            _logger.debug('remount?? %s', v)
```
Comment (ejannett) [inline lib/oci_utils/migrate/imgdevice.py:648]: can we have only one call ?

File: lib/oci_utils/migrate/imgdevice.py [hunk 9]
```diff
@@ -1381,19 +1249,13 @@ def get_os_release(self):
...
                     break
-                else:
-                    _logger.debug('os-release not found in %s' % mnt)
+                _logger.debug('os-release not found in %s', mnt)
         except Exception as e:
-            _logger.error('   Failed to collect os data: %s' % str(e),
-                          exc_info=True)
-        _logger.debug('os data: %s' % osdict)
+            _logger.error('   Failed to collect os data: %s', str(e), exc_info=True)
+        _logger.debug('os data: %s', osdict)
```
Comment (ejannett) [inline lib/oci_utils/migrate/imgdevice.py:1258]: Are you sure the stack trace is for the 'error' call , not the 'debug' one ?

File: buildrpm/oci-utils.spec [hunk 10]
```diff
@@ -1,5 +1,5 @@
 Name: oci-utils
-Version: 0.11.6
+Version: 0.11.7
 Release: 6%{?dist}
```
Comment (ejannett) [inline buildrpm/oci-utils.spec:3]: should it be '7' also ?

**Review by ejannett — APPROVED** [review]

File: lib/oci_utils/impl/migrate/oci-image-migrate-import-main.py (hunk 1 above)
Comment (guidotijskens) [reply lib/oci_utils/impl/migrate/oci-image-migrate-import-main.py:29]: right, this is a mess
changed

File: lib/oci_utils/impl/migrate/oci-image-migrate-upload-main.py (hunk 2 above)
Comment (guidotijskens) [reply lib/oci_utils/impl/migrate/oci-image-migrate-upload-main.py:133]: deleted

File: lib/oci_utils/migrate/__init__.py (hunk 3 above)
Comment (guidotijskens) [reply lib/oci_utils/migrate/__init__.py:22]: changed

File: lib/oci_utils/migrate/__init__.py (hunk 4 above)
Comment (guidotijskens) [reply lib/oci_utils/migrate/__init__.py:237]: changed

File: lib/oci_utils/migrate/__init__.py (hunk 5 above)
Comment (guidotijskens) [reply lib/oci_utils/migrate/__init__.py:274]: no, it is useless here

File: lib/oci_utils/migrate/image_types/qcow2.py (hunk 6 above)
Comment (guidotijskens) [reply lib/oci_utils/migrate/image_types/qcow2.py:95]: changed

File: lib/oci_utils/migrate/image_types/vmdk.py (hunk 7 above)
Comment (guidotijskens) [reply lib/oci_utils/migrate/image_types/vmdk.py:297]: this is to get the attention of the operator of possible issues later, i.e. at instance creation and to make sure the message is read.

File: lib/oci_utils/migrate/imgdevice.py (hunk 8 above)
Comment (guidotijskens) [reply lib/oci_utils/migrate/imgdevice.py:648]: changed

File: lib/oci_utils/migrate/imgdevice.py (hunk 9 above)
Comment (guidotijskens) [reply lib/oci_utils/migrate/imgdevice.py:1258]: no, but we do not need the stack trace here at all, only the info it failed and why

**Pull request merged**